File: Java/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/manzanoPag46/ExercicioI.java
package manzanoPag46;

import java.util.Scanner;

/*i) Elaborar um programa que efetue a leitura de 10 valores numéricos e apresente no final o total do
somatório e a média aritmética dos valores lidos. */

public class ExercicioI {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		int valor1 = 0;
		int soma = 0;
		float mediaAritmetica = 0;
		int contadora = 0;

		while (contadora < 10) {
			contadora = contadora + 1;
			System.out.println("Digite o " + contadora + " valor:");
			valor1 = sc.nextInt();
			soma = soma + valor1;
		}

		mediaAritmetica = soma / 10;

		System.out.println("A soma de todos valores é: " + soma);
		System.out.println("A média Aritmética dos números é: " + mediaAritmetica);

		sc.close();

	}

}

File: LogicaDeProgramacao(JavaScript)/exercicios pag 46(Manzano)/js/exercicioI.js
/*i) Elaborar um programa que efetue a leitura de 10 valores numéricos e apresente no final o total do
somatório e a média aritmética dos valores lidos. */

var valor1 = 0
var soma = 0
var mediaAritmetica = 0
var contadora = 0

while (contadora < 10) {
    contadora = contadora + 1
    valor1 = parseInt(prompt("Digite o "+contadora+" valor:"))
    soma = soma + valor1    
}

mediaAritmetica = soma / 10

alert("A soma de todos valores é: "+soma)
alert("A média Aritmética dos números é: "+mediaAritmetica)